.github/workflows/jrgitlint.yml

```diff
@@ -0,0 +1,19 @@
+name: JR GitLint
+
+on: [push, pull_request]
```

Comment:
I don't think you want 'push' here.

GITHUB_BASE_REF and GITHUB_HEAD_REF are only defined for the pull-request event.
https://docs.github.com/en/actions/reference/events-that-trigger-workflows#pull_request